cdc/api/v2/model.go

```diff
@@ -413,6 +413,16 @@ func (c *ReplicaConfig) toInternalReplicaConfigWithOriginConfig(
 			res.Sink.AdvanceTimeoutInSec = util.AddressOf(*c.Sink.AdvanceTimeoutInSec)
 		}
 
+		if c.Sink.GlueSchemaRegistryConfig != nil {
```

Comment:
It loos this config only belongs to the kafka sink ? so put it inside the `kafkaConfig` ?

Reply:
done